Kickstarter-iOS/Views/Controllers/ProjectPamphletViewController.swift

```diff
@@ -186,6 +206,50 @@ public final class ProjectPamphletViewController: UIViewController {
     self.present(vc, animated: true)
   }
 
+  private func goToManagePledge(project: Project, reward: Reward, refTag _: RefTag?) {
+    let managePledgeViewController = ManagePledgeViewController.instantiate()
+    managePledgeViewController.configureWith(project: project, reward: reward)
+
+    let nav = UINavigationController(rootViewController: managePledgeViewController)
+    if AppEnvironment.current.device.userInterfaceIdiom == .pad {
+      _ = nav
+        |> \.modalPresentationStyle .~ .formSheet
+    }
+    self.present(nav, animated: true)
+  }
+
+  private func goToDeprecatedPledge(project: Project, reward: Reward, refTag _: RefTag?) {
+    let pledgeViewController = DeprecatedRewardPledgeViewController
+      .configuredWith(
+        project: project, reward: reward
+      )
+    pledgeViewController.navigationItem.leftBarButtonItem = UIBarButtonItem(
```

Comment:
Is there a reason we've decided to set this externally on `DeprecatedRewardPledgeViewController`?

If we do this inside of the controller and set `self.navigationItem` (like we do everywhere else we will be able to remove this code as well the identical code in `ProjectPamphletContentViewController`.

But maybe I'm missing the reason here? 🤔

Reply:
On a rush to unblock the other tickets I had copied the code from somewhere else. Good catch.